changelog.md

```diff
@@ -1,3 +1,5 @@
+2024-12-10 - 7f771ad53e40a51c4394ffa76812cb062d9226c7 - Make select disable of non-deletable items in oxd-list
```

Comment:
Please follow the standard format

[date] - [commit hash] - [relative path] - [change]